integration_test/tests/mongosqltranslate_tests.rs

```diff
@@ -0,0 +1,157 @@
+mod common;
+
+mod mongosqltranslate {
+    use crate::common::{
+        allocate_env, connect_with_conn_string, default_setup_connect_and_alloc_stmt,
+        disconnect_and_close_handles, fetch_and_get_data, get_column_attributes,
+        get_sql_diagnostics,
+    };
+    use cstr::WideChar;
+    use definitions::{
+        AttrOdbcVersion, CDataType, HStmt, Handle, HandleType, SQLExecDirectW, SQLExecute,
+        SQLPrepareW, SqlReturn, SQL_NTS,
+    };
+
+    #[test]
+    fn test_srv_style_uri_connection() {
```

Comment:
This test requires libmongosqltranslate, so I moved it here to make testing easier.